src/main/java/org/fisco/bcos/web3j/protocol/core/Ethereum.java

```diff
@@ -1,131 +1,40 @@
 package org.fisco.bcos.web3j.protocol.core;
 
 import java.math.BigInteger;
-import java.util.List;
-
-import org.fisco.bcos.web3j.protocol.core.methods.request.ShhFilter;
-import org.fisco.bcos.web3j.protocol.core.methods.request.ProofMerkle;
-//增加eth_pbftView接口
 import org.fisco.bcos.web3j.protocol.core.methods.response.*;
 
 /**
- * Core Ethereum JSON-RPC API.
+ * Core FISCO BCOS JSON-RPC API.
  */
 public interface Ethereum {
     Request<?, Web3ClientVersion> web3ClientVersion();
 
-    Request<?, Web3Sha3> web3Sha3(String data);
-
-    Request<?, NetVersion> netVersion();
-
-    Request<?, NetListening> netListening();
-
-    //Request<?, NetPeerCount> netPeerCount();
-
-    Request<?, EthProtocolVersion> ethProtocolVersion();
-
-    Request<?, EthCoinbase> ethCoinbase();
-
     Request<?, EthSyncing> ethSyncing();
 
-    Request<?, EthMining> ethMining();
-
-    Request<?, EthHashrate> ethHashrate();
-
-    Request<?, EthGasPrice> ethGasPrice();
-
-    Request<?, EthAccounts> ethAccounts();
-
     Request<?, EthBlockNumber> ethBlockNumber();
     
-    //增加eth_pbftView接口
     Request<?, EthPbftView> ethPbftView();
     
-    //增加getMinerList接口
     Request<?, MinerList> getMinerList();
     
-    //增加getObserverList接口
     Request<?, ObserverList> getObserverList();
 
-    //增加consensusStatus接口
     Request<?,EthConsensusStatus> consensusStatus();
 
-    //增加groupList接口
     Request<?, GroupList> ethGroupList();
```

Comment:
do not use ethXXX, use our api directly

Reply:
I remove some useless eth api in the commit, the next commit will modify name for our useful api.